stats.tex

```diff
@@ -0,0 +1,50 @@
+%% Stats
+The \verb|sympy.stats| module provides random variable types and methods for
+determination of statistical properties of expressions involving random
```

Comment:
_statistical properties_ is a bit ambiguous. What _sympy.stats_ currently does, is to generate the symbolic integrals/summations for statistical expressions. No properties are determined, symbolic calculations are performed.